src/Microsoft.ML.AutoML/API/RankingExperiment.cs

```diff
@@ -68,10 +78,11 @@ public static class RankingExperimentResultExtensions
         /// </summary>
         /// <param name="results">Enumeration of AutoML experiment run results.</param>
         /// <param name="metric">Metric to consider when selecting the best run.</param>
+        /// <param name="dcgTruncationLevel">Maximum truncation level for computing (N)DCG. Defaults to 3.</param>
         /// <returns>The best experiment run.</returns>
-        public static RunDetail<RankingMetrics> Best(this IEnumerable<RunDetail<RankingMetrics>> results, RankingMetric metric = RankingMetric.Ndcg)
+        public static RunDetail<RankingMetrics> Best(this IEnumerable<RunDetail<RankingMetrics>> results, RankingMetric metric = RankingMetric.Ndcg, int dcgTruncationLevel = 3)
```

Comment:
@michaelgsharp: Do you recall why this is set to `3` instead of `10`? The rest defaults to `10`.

Reply:
I bet that I just missed that one when I was making changes. It should be 10 to match all the others. I'll fix it.

Reply:
Fixed.